akka-actor-typed-tests/src/test/scala/akka/actor/typed/scaladsl/adapter/AdapterSpec.scala

```diff
@@ -162,9 +162,11 @@ object AdapterSpec {
 
 }
 
-class AdapterSpec extends AkkaSpec {
+class AdapterSpec extends WordSpec with Matchers with BeforeAndAfterAll with WithLogCapturing {
```

Comment:
not using the Typed `with LogCapturing`?

Reply:
Dang!